### ISSUE TITLE
Loading indicators, inline and modal, should announce status to screen readers

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#7722]
Form Runner's modal and inline loading indicators are visual only, so screen reader users might not be informed that an action is in progress after activating a process button.

[+1 from customer](https://app.basecamp.com/3600924/buckets/7993860/messages/9949706804)
